README.md

```diff
@@ -50,7 +50,7 @@ Handlers are matched in order. If no handler applies to a deprecation
 warning, Ruby warnings is ignored, Rails warnings are passed through
 the ActiveSupport notification mechanism.
 
-The following sample logs all deprecation warnings:
+The following example logs all deprecation warnings:
```

Comment:
I think "example" makes more sense here. It used throughout the README for other handler snippets.